packages/muelu/src/MueCentral/MueLu_VerboseObject.cpp

```diff
@@ -140,13 +130,18 @@ namespace MueLu {
     return globalVerbLevel_;
   }
 
-  void VerboseObject::SetDefaultOStream(const Teuchos::RCP<Teuchos::FancyOStream>& defaultOStream) {
-    defaultOStream->setOutputToRootOnly(-1);
-    setDefaultOStream(defaultOStream);
+  void VerboseObject::SetMueLuOStream(const Teuchos::RCP<Teuchos::FancyOStream>& mueluOStream) {
+    mueluOStream->setOutputToRootOnly(-1);
+    GetMueLuOStream() = mueluOStream;
```

Comment:
@cgcgcg What does this construct do?

Reply:
@jhux2 It just sets the stream. The reason to do it this way is because it is stored (and returned) as a static variable in that function.

Reply:
@aprokop Thanks.  I'd never seen this before.